cnames_active.js

```diff
@@ -1918,6 +1918,7 @@ var cnames_active = {
   "service-mocker": "service-mocker.github.io/service-mocker",
   "serviceberry": "bob-gray.github.io/serviceberry",
   "sesam": "lennertderyck.github.io/sesam",
+  "socket": "https://oyuncubro01.github.io/logger",
```

Comment:
```suggestion
  "socket": "oyuncubro01.github.io/logger",
```
`https://` is not needed